canonical/ubuntu.com
# [Sunbeam GA Launch] WD-3611 update openstack section on `/legal/ubuntu-pro-description#uprosd-openstack`

## Done

- Update the OpenStack section on `/legal/ubuntu-pro-description#uprosd-openstack`

## QA

- Check out the [demo here](url) and compare the provided [copy doc](https://docs.google.com/document/d/1pPMRLYlWS1Fs12ftt4KsUgYmFDf0DBba04Vl40IBW3U/edit#).
  - This doc isn't a regular copy doc, as the actual copy is in a bit of a mess, so this doc was provided just specifically for this section.

## Issue / Card

- [WD-3611](https://warthogs.atlassian.net/browse/WD-3611)

## Help

[QA steps](https://discourse.canonical.com/t/qa-steps/152) - [Commit guidelines](https://discourse.canonical.com/t/commit-guidelines/148)
